health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/components/SelectingBoundariesDuplicate.js

```
@@ -77,52 +89,62 @@ const SelectingBoundariesDuplicate = ({ onSelect, formData, ...props }) => {
+              boundaryOptions={boundaryOptions}
+              updateBoundary={updateBoundary}
+              hierarchyData={props?.props?.hierarchyData}
+              isMultiSelect={"true"}
+              //   onSelect={(value) => {
+              //     setSelectedData(value?.selectedData);
+              //     setBoundaryOptions(value?.boundaryOptions);
+              //   }}
+              onSelect={(value) => {
+                handleBoundaryChange(value);
+              }}
+            ></Wrapper>
```

Biome

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)